test/system/005-info.bats

```diff
@@ -182,6 +182,17 @@ host.slirp4netns.executable | $expr_path
     is "$output" ".*graphOptions: {}" "output includes graphOptions: {}"
 }
 
+@test "podman info - additional image stores" {
+    skip_if_remote "--storage-opt flag is not supported for remote"
+    store1=$PODMAN_TMPDIR/store1
+    store2=$PODMAN_TMPDIR/store2
+    mkdir -p $store1 $store2
+    run_podman info --storage-opt=$(podman_storage_driver)'.imagestore='$store1 \
+                    --storage-opt=$(podman_storage_driver)'.imagestore='$store2 \
+                    --format '{{index .Store.GraphOptions "'$(podman_storage_driver)'.additionalImageStores"}}'
```

Comment:
```
{{index .Store.GraphOptions "'$(podman_storage_driver)'.additionalImageStores"}}\n{{index .Store.GraphOptions "'$(podman_storage_driver)'.imagestore"}}
```
Can you use this format then check 
```
assert "${lines[0]}" == "["$store1" "$store2"]" "output includes additional image stores"
assert "${lines[1]}" == "$store2" "old imagestore output"
```

In general `assert` should be preferred over `is` as `is` has some rather dangerous pitfalls, i.e. your current test can end up broken as `[]` is valid regex so is may assume it should perfom a regex match not an exact match.
With `assert` you must specify `==` or `=~` so there is no ambiguity

Reply:
Updated